### DISCUSSION TITLE
1.27.0

### DISCUSSION THREAD

**Discussion by xaoxuu** [xaoxuu/hexo-theme-stellar#400] [Announcements]
**新特性**

- 新增右侧栏，原 `sidebar` 更名为 `leftbar`

**重要变更**

- toc 样式修改为适应右边栏，不建议放在左边栏了
- plugins 插件重构
- 内置服务配置转移到 `data_services ` 中
- 配置中的替换符号 `${xxx}` 由于存在冲突，格式更改为 `{xxx}`
- Waline 评论插件升级到 3.x 

**修复**

- 修复了 macOS 系统上的 `.DS_Store` 文件导致报错的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/xaoxuu/hexo-theme-stellar/releases/tag/1.27.0'>1.27.0</a>.</em>

**Comment by yeelvus** [comment]
最近两天正好发现在桌面端两侧太空旷了，你现在就加入了右侧栏，很开心。
